src/leader.py

```diff
@@ -63,13 +63,21 @@ def turn_straight(self):
         self.fw.turn(self.STRAIGHT_ANGLE)
 
 
-    def turn(self,position):
-        if position == -1:
-            self.turn_left()
-        elif position == 1:
-            self.turn_right()
-        else:
-            self.turn_straight()
+    def set_turn_angle(self, position):
+        self.fw.turn(self.STRAIGHT_ANGLE + (self.MAX_TURN_ANGLE  * position))
```

Comment:
This function needs a comment. I know what it's doing because I wrote the InputController, but no one would know just by looking at it.

Reply:
Actually, just scrap this function. You're not setting the turn angle. you're turning the car itself. Just use this line of code instead of calling the function. We're not in it for LOC here.